cmd/lncli/commands.go

```diff
@@ -3532,7 +3532,7 @@ var forwardingHistoryCommand = cli.Command{
 	payment circuits (HTLCs) over a particular time range (--start_time and
 	--end_time). The start and end times are meant to be expressed in
 	seconds since the Unix epoch. If --start_time isn't provided,
-	then 24 hours ago is used.  If --end_time isn't provided,
+	then the Unix epoch (01-01-1970) is used.  If --end_time isn't provided,
```

Comment:
this change can be removed

Reply:
That text change was reverted, do you want me to squash commits maybe?

Reply:
Just make one commit for `rpcserver` and one for `lncli` :)